src/test/resources/outputs/trino/basics/exclude.sql

```diff
@@ -5,16 +5,16 @@ SELECT "t"."flds" AS "flds" FROM "default"."EXCLUDE_T" AS "t";
 SELECT "t"."foo" AS "foo" FROM "default"."EXCLUDE_T" AS "t";
 
 --#[exclude-02]
-SELECT "$__EXCLUDE_ALIAS__"."t"."flds" AS "flds", "$__EXCLUDE_ALIAS__"."t"."foo" AS "foo" FROM (SELECT (SELECT (SELECT (SELECT "t"."flds"."a"."field_x" AS field_x, "t"."flds"."a"."field_y" AS field_y) AS a, (SELECT "t"."flds"."c"."field_x" AS field_x, "t"."flds"."c"."field_y" AS field_y) AS c) AS flds, "t"."foo" AS foo) AS "t" FROM "default"."EXCLUDE_T" AS "t") AS "$__EXCLUDE_ALIAS__";
+SELECT "$__EXCLUDE_ALIAS__"."t"."flds" AS "flds", "$__EXCLUDE_ALIAS__"."t"."foo" AS "foo" FROM (SELECT CAST(ROW(CAST(ROW(CAST(ROW("t"."flds"."a"."field_x", "t"."flds"."a"."field_y") AS ROW(field_x INTEGER, field_y VARCHAR)), CAST(ROW("t"."flds"."c"."field_x", "t"."flds"."c"."field_y") AS ROW(field_x INTEGER, field_y VARCHAR))) AS ROW(a ROW(field_x INTEGER, field_y VARCHAR), c ROW(field_x INTEGER, field_y VARCHAR))), "t"."foo") AS ROW(flds ROW(a ROW(field_x INTEGER, field_y VARCHAR), c ROW(field_x INTEGER, field_y VARCHAR)), foo VARCHAR)) AS "t" FROM "default"."EXCLUDE_T" AS "t") AS "$__EXCLUDE_ALIAS__";
```

Comment:
(self-review) prior tests work as before. I'm planning to take on https://github.com/partiql/partiql-scribe/issues/60, which should simplify the output transpiled queries for Trino and Spark